packages/frontend/component/src/lit-react/create-component.ts

```diff
@@ -318,14 +316,13 @@ export const createComponent = <
 
     React.useLayoutEffect(() => {
       const container = containerRef.current;
-      if (!container) {
+      const element = elementRef.current;
+      if (!container || !element) {
         return;
       }
+      if (element.isConnected) return;
       container.append(element);
-      return () => {
-        element.remove();
-      };
-    }, [element]);
```

Comment:
@pengx17 PTAL

According to the comment in `disconnectedCallback()`, it should ensure that nothing holds a reference to the element. Therefore, in React's strict mode, the element will `connect`, `disconnect`, and then `connect` again sequentially. The later `connect` is an invalid operation where `EditorHost` is `null`, causing some issues that make the CI fail.

call timing sequence：

![image](https://github.com/user-attachments/assets/953d7d9d-ec99-40f0-8e08-6603557b1ed9)
1. element.append
  a. std.render
    1. host.connect
      a. std.mount
2. element.remove
  a. host.disconnect
    1. std.unmount
3. element.append
  a. host.connect
    1. std.mount , in this moment `std.host === null`, which make CI failed

Reply:
Cool! I